packages/teams-js/src/public/sharing.ts

```diff
@@ -5,6 +5,10 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { ErrorCode, SdkError } from './interfaces';
 import { runtime } from './runtime';
 
+/**
+ * Namespace to open a share dialog for web content.
+ * For more info, see {@link https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab}
```

Comment:
Let's add link text (article title) to give customers more context about what it leads to:

{@link https://learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/share-to-teams-from-personal-app-or-tab **Share to Teams from personal app or tab**}

Reply:
Updated in the latest commit.